mailpoet/lib/Automation/Engine/Builder/UpdateAutomationController.php

```diff
@@ -66,6 +76,10 @@ public function updateAutomation(int $id, array $data): Automation {
       }
     }
 
+    if ($automation->getStatus() === Automation::STATUS_DRAFT) {
+      $this->unscheduleAutomationRuns($automation);
+    }
```

Comment:
Maybe we could check for something like `$automation->getStatus() === Automation::STATUS_ACTIVE && $data['status'] ===  Automation::STATUS_DRAFT` earlier in the method, so that we don't run the query for every update, when users are editing their automations?